modepy/quadrature/__init__.py

```diff
@@ -167,6 +173,8 @@ def __init__(self, quads: Iterable[Quadrature]) -> None:
 
         super().__init__(x, w, exact_to=exact_to)
 
+        self.quadratures = quads
+
```

Comment:
I just don't like it. It feels brittle. Also, does mypy fish type annotations out of the constructor?j

Reply:
Fair enough, I'm not a big fan either because there's other things happening in the constructor.

`mypy` should catch it though, since it's just a normal instance attribute. It might need to explicitly add the type, i.e. `self.quadratures: Sequence[Quadrature] = quads`, to make it all work nicely for both sphinx and mypy.